### DISCUSSION TITLE
MMORPG character stats

### DISCUSSION THREAD

**Discussion by TienDat2k** [aieuo/Mineflow#126] [General]
Hi. It's been a while since I asked you a few questions and now I come back and continue with a new question :D. I am a huge fan of MMORPG but when it comes to work. then I have an important problem that is the index. This greatly determines whether a game is good or not when creating it and optimizing it. Currently, I don't know how to create MANA to apply to Skill and when using Skill, MANA will be deducted, over time MANA will recover a certain amount (It is similar to the character's healing when you are full) or when you use another item made to replenish MANA. And the same goes for stats like HEALTH, DEF ATK, SPEED. . . and how to extract the whole information to generate the character state table? .Sorry to bother you, please give me some suggestions or instructions on how to create so that I can continue with your work. And this is the most amazing plugin i know, its extremely customizable as long as i have an idea i can do anything with it,if there is a better paid version i am willing to pay for it. buy it. And if there is a course on how to use it better, that would be great

**Comment by aieuo** [comment]
Thank you for using this plugin.  
I think you can use actions of the Config category.
I made an [example recipe](https://github.com/aieuo/MineflowExamples/blob/master/Counter.json)  that uses those actions.

**Reply by TienDat2k** [reply]
Thank you so much

**Reply by TienDat2k** [reply]
I translated the plugin into Vietnamese from last week, based on the Japanese language and translated it, is there any way I can contribute the translation?. I really want the next updates to have Vietnamese language for use ease.

**Reply by aieuo** [reply]
Please fork this repository and create `vie.ini` into the [`resources/language`](https://github.com/aieuo/Mineflow/tree/master/resources/language) folder and then create a pull request.  
I appreciate your translation!
